News week Clone Page

smcommits opened the pull request:
This project is contains the clone of [the home page](https://www.newsweek.com/) News Week and I am creating a pull request to get a review on the code. 

The aim of this project was to clone website using bootstrap as much as possible. 

As required, I have made a heavy use of bootstrap framework. 
I have also created some custom mixins like bottom, right, left, and top position. 
A mixin for custom font sizing using 1 em as a base size. 
I have also created customized color variables of bootstrap framework to match it with the webstie. 

I have added the linters webhint and stylelint to fix basic issues. 

Please feel free to provide any feedback!

smcommits committed:
Initalize index.html and progress with section

Initiaized index.html and added different sections while simultaneously
providing bootstrap classes

Created a custom mixin for bootstrap

Sections Added
- Featured Stories
Added 5 sub sections
Used custom _top-right mixin to postion overlay text
Used figure for image and h5 and p for text
Used to respective bootstrap classes for styling

- Top Story
Added 1 sub sections
Used custom _top-right mixin to postion overlay text
Used figure for image and h5 and p for text
Used to respective bootstrap classes for styling

- Sponsered Section
Added 5 sub sections
Used custom _top-right mixin to postion overlay text
Used bootstrap class 'Card' on the parent and respective classes on
children

- My turn and Culture Section
Added 3 sub sections to each
Used bootstrap classes for styling

smcommits committed:
Update HTML, JS, Bootstrap

**HTML
Sections Added
- More Stories
Added More Stories section and styled using bootstrap media object
- Sponored 2
Added the second sponsor section containing sign-up form
- The Debate
Added the debate section and opinion section and styled them using
bootstrap media object
-Bottom Section
Added the bottom section with Magazine section and editor's pick section
and styled then using bootstrap classes

**Bootstrap
Added a bootstrap in three sectoins to create a grid layout

**JS
Added script to move elements on window resize

smcommits committed:
Add HTML and Bootstrap

HTML
- Added Newsletter section
- Added Subscribe section

Bootstrap
- Used row/column to style subscribe section
- Used absolute position to style magazine image in subscribe section
- Styled Newsletter using column/row

smcommits committed:
Update HTML and Bootstrap

- Fixed the footer nav
- Fixed padding for the magazine section
- Fixed the 'World' section and replicated it 8 times
- Added top navbars for large screen sizes

smcommits committed:
Deleted unwanted file bootstrap.css.map

smcommits committed:
Add screenshot of webpage

smcommits committed:
Format README file

Added Project Name and Description
Added the screenshot 
Added the 'built with' section 
Added a live demo 
Added getting started
Added autor's detail

smcommits committed:
Fixed screenshot location in README

smcommits committed:
Fix screenshot location in README

smcommits committed:
Update .gitingnore and .github

Added the linter file to .gitignore
Added .github folder which contains linter files for the github

smcommits committed:
Merge branch 'news-week-clone' of https://github.com/smcommits/News-Week into news-week-clone

smcommits committed:
Fix padding on subscribe section

- Changed the padding on subscribe section from 5 to 2

jubaan reviewed (changes requested):
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @smcommits  . Thanks for submitting you project!

Things to notice:
:100: Descriptive PR
:100: Descriptive README
:100: Awesome solid start!
:100: Great sticky effect

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] remove the request to files that are non-existent
![image](https://user-images.githubusercontent.com/25774518/95888799-2b95aa80-0d47-11eb-9953-708ad32abbcf.png)

- [ ] It is prefefer to use the CDN bootstrap link to use bootstrap

- [ ] Keep consistency in the alignment and indentation in HTML file

- [ ] Use semantic HTML for the ```<articles>``` instead of ```<div>``` containers

- [ ] Issues link in README broken.

- [ ] Missing link in editors pick element 
![image](https://user-images.githubusercontent.com/25774518/95887537-9645e680-0d45-11eb-8b92-bebc2f558697.png)

- [ ] Incorrect alignment in newsletter element for text in desktop
![image](https://user-images.githubusercontent.com/25774518/95887667-c097a400-0d45-11eb-9fd4-81dc62a53494.png)

- [ ] footer is missing elements
![image](https://user-images.githubusercontent.com/25774518/95887768-dad18200-0d45-11eb-9bf6-cb92130c7a1c.png)

- [ ] Elements not displaying properly in tablet mode
![image](https://user-images.githubusercontent.com/25774518/95887857-f3da3300-0d45-11eb-953b-3a025acc0db7.png)

- [ ] In the magazine element don't match the original's style
![image](https://user-images.githubusercontent.com/25774518/95888020-2e43d000-0d46-11eb-82f8-d703f17b1923.png)

- [ ] Footer don;t look as original in tablet mode
![image](https://user-images.githubusercontent.com/25774518/95888108-4c113500-0d46-11eb-82b3-f51a93d717db.png)

- [ ] Navbar doesn't look as expected in mobile view
![image](https://user-images.githubusercontent.com/25774518/95888172-5e8b6e80-0d46-11eb-9f21-6916d0ab3ea9.png)

- [ ] Element doesn't look as original
![image](https://user-images.githubusercontent.com/25774518/95888243-74009880-0d46-11eb-9a3b-eb7e4c438588.png)

- [ ] Subscribe section doesn't look like the original
![image](https://user-images.githubusercontent.com/25774518/95888349-998da200-0d46-11eb-9310-ea562740dc42.png)

- [ ] footer doesn't look like the original in mobile
![image](https://user-images.githubusercontent.com/25774518/95888409-ad390880-0d46-11eb-873b-34c79e587096.png)

### [OPTIONAL]  :yellow_circle:  

- [ ] Add other social network to the author section of the README.

### [Comments] :green_circle: 

- A great tool to test responsiveness in different screen sizes is the use of the browser media query inspector. Use it to compare the clone against the original.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

smcommits committed:
Update index.html

smcommits committed:
Update reveiw requests

Fixed the alignment of magazine section
Fixed the alignment of newsletter section
Added Footer bottom
Fixed upper footer view on mobile screen
Fixed overflow on tablet and mobile view

smcommits committed:
Update change request

Replaced div tags with article tag for semantic design
Added the align effect to in the magazine section to match the original
style

smcommits committed:
Merge branch 'news-week-clone' of https://github.com/smcommits/News-Week into news-week-clone

smcommits committed:
Resolve git conflicts

Pull and merged the changes

smcommits committed:
Fix webhint errors

- Fixed unclosed tag error
- Added headings to the articles

smcommits committed:
Update README.md

Rhelli reviewed (changes requested):
# Changes Required! 🛑 ♻️

Hi @smcommits, great work on the project so far, I'm extremely impressed at your complete use of bootstrap and absolutely no CSS 👀 . Amazing Job! There are, however, a few minor changes that need to be made before this project can be approved. Lets get them sorted to so you can move onto the next project!

********

**[Mandatory]** - You appear to have a duplicated section ('Sponsered Insights') appearing on tablet size screens, coming immediately after the opinion section. See the images below for reference:

**Your version of Newsweek** 👇🏾
![image](https://user-images.githubusercontent.com/48107460/95995097-f6618900-0e28-11eb-8cd6-91c82575e2a2.png)

**Newsweeks own website** 👇🏾
![image](https://user-images.githubusercontent.com/48107460/95995143-05483b80-0e29-11eb-8dd6-10702b7f21b5.png)

******

2. **[Mandatory]** - This may be a part of the problem above, however, you also appear to be missing the 'Free Sign Up' section that should appear at the end of the 'Opinion' section and before the 'In the Magazine' section.  This occurs over all screen sizes. Please see the screenshots below for reference.

**Your version of Newsweek**👇🏾
![image](https://user-images.githubusercontent.com/48107460/95996164-468d1b00-0e2a-11eb-8b0f-6b5c08de5481.png)


**Newsweeks version** 👇🏾
![image](https://user-images.githubusercontent.com/48107460/95995452-67a13c00-0e29-11eb-87e3-28bcc44761ba.png)

****

3. **[Mandatory]** - Similar to the first issue here, on mobile screens, the 'Sponsored Insight' section is duplicated on mobile where it should lead straight into the 'In The Magazine Section'

**Your version ** 👇🏾
![image](https://user-images.githubusercontent.com/48107460/95995758-d41c3b00-0e29-11eb-83ac-4a283faa6a39.png)

**The Newsweek Version** 👇🏾
![image](https://user-images.githubusercontent.com/48107460/95995806-df6f6680-0e29-11eb-8008-d3b768c53375.png)

******

Get those issues fixed and we'll be able to get your project approved! Again, well done on the project so far - it's a rarity too see absolutely **no** CSS on this project, so cudos to you for getting the most out of Bootstrap!

If you've got any questions or need any help, @ me here on Github or message me in Slack.

All the best,

Rory

Rhelli reviewed (approved):
# Approved! 🚀✅

Well done on amending the changes made my previous TSEs. On reflection on my previous amendments required and the link you shared with me to the version of the [Newsweek website from the 10th Oct](https://web.archive.org/web/20201010073812/https://www.newsweek.com/), it appears you have made all necessery changes and made a true-to-life Newsweek replica!

Keep up the good work!

All the best,

Rory
